### DISCUSSION TITLE
STC15F receive the ROM command using one wire communication

### DISCUSSION THREAD

**Discussion by karthickrajapetal** [grigorig/stcgal#93] [Q&A] [answered]
Hai Sir,
 I'm using STC15Fxxx series , I'm try to read the Read ROM Command using one wire communication in STC15F104 but i can't it.
 kindly provide the any sample program to read the ROM Command or receive the command in one wire communication .
 
Thanks ,
M.Karthickraja.

**Comment by area-8051** [comment, marked as answer]
This is unrelated to STCGAL, issue closed.

**Reply by karthickrajapetal** [reply]
Ok sir, Is it possible to read the HEX file in STC15F?

**Reply by area-8051** [reply]
No, there's no provision for this in the chip.

Theoretically, you could write a program to dump flash memory through the serial port, but this program would overwrite the first page of the flash, so there's no real interest in doing this.
